### ISSUE TITLE
If an html attribute value starts with 'view' the closing double quote is not recognized

### ISSUE THREAD

**Issue by kraussh** [Microsoft/vscode#13949]
- VSCode Version: Code 1.6.1 (9e4e44c19e393803e2b05fe2323cf4ed7e36880e, 2016-10-13T16:21:53.542Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: ErikLieben.aurelia-0.1.4, felixfbecker.php-debug-1.9.4, ms-vscode.csharp-1.4.1, redhat.java-0.0.4, Telerik.nativescript-0.3.1, vsmobile.cordova-tools-1.2.3

Steps to Reproduce:

1. Create an HTML file
2. Put the following text in:
<html>
    <head>
        <meta name="viewport">
    </head>
</html>
![vscodebug](https://cloud.githubusercontent.com/assets/13222616/19488002/dc521e12-9519-11e6-8805-6c9bc6b247ff.png)

In the above html everything after the second double quote in <meta name="viewport"> will be colored as an attribute value.  This only happens if the attribute value starts with the text 'view' but not if it is only the text 'view'.

**Comment by aeschli** [comment]
This goy fixed for October.
![image](https://cloud.githubusercontent.com/assets/6461412/19509037/d9588cfa-95dc-11e6-8aee-35573a1b98f8.png)

**Comment by kraussh** [comment]
Hi Everyone - I've upgraded to VSCode 1.7.1 but still experiencing this issue.
![htmlhighlighting](https://cloud.githubusercontent.com/assets/13222616/19981587/c7467c18-a1bf-11e6-965f-4da738068135.png)

**Comment by aeschli** [comment]
@kraussh I'm not able to reproduce that on 1.7.1. Can you file a new issue that includes the code sample (not just the screenshot)? Thanks a lot!
![image](https://cloud.githubusercontent.com/assets/6461412/19998225/01cb9ae2-a26c-11e6-9c15-eeb5a30d0ebe.png)

**Comment by kraussh** [comment]
My apologies.  The issue was caused by the version of the aurelia extension I had installed.  I updated the aurelia version from 0.1.4 to 0.1.10 and problem went away.  Before and after screens below.  Let me know if I should still file a new issue.
![before](https://cloud.githubusercontent.com/assets/13222616/20193337/6220ba50-a742-11e6-89cb-464df9ca4b6e.png)
![after](https://cloud.githubusercontent.com/assets/13222616/20193347/6788d388-a742-11e6-99b1-6f83be77e63e.png)
